umbraco/The-Starter-Kit
# Fix tours

## Details
- Fixed the "_Create content_" and "_Data structure_" tours
  - Since we introduced back tabs the DOM changed, therefore the tours broke
- Updated the version to the latest 8.17
- Change the SK package zip version

## Test
- Build the package according to [this](https://github.com/umbraco/The-Starter-Kit#building-the-package)
- Download the latest [v8 release](https://our.umbraco.com/download/releases/8171)
- Make sure you install Umbraco without the SK ❗ 
- Go to the **Packages** section > **Install local** > drop the SK **.zip** from the `build\Releases` folder
- Complete all the tours 
  - Create content tour > The Media Library > 9th step (The uploaded image) doesn't highlight nicely because this element has no default size, so this is the best me and Mads could figure our 🤷 
</br>

- Also try to start the SK in this project and make sure it still works after the update
---
Note: ❗ 
- The fix is within the **create-content.json**, **data-structure.json** and **build.xml**
- Changes to *.config happened because of the NuGet packages updates (they prompted to be overwritten)
- The rest of the changes are as a result of the _UmbracoForms_, _Umbraco.ModelsBuilder_ and _UmbracoCms.Core_ updates
- You have to regenerate the models when running the Web project (Settings > Models Builder)
